### PR TITLE
added delete reminder command and test cases

### PR DESCRIPTION
**Pull request by minpyaemoe**
Aded delete reminder command and test cases

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java [hunk 1]
```diff
@@ -1,49 +1,24 @@
...
 /**
  * Adds a Reminder to the Reminder list.
  */
-public class ReminderCommand extends Command {
+public abstract class ReminderCommand extends Command {
 
     public static final String COMMAND_WORD = "reminder";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Reminder to MoneyGoWhere. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " " + AddReminderCommand.COMMAND_WORD
```
Comment (Nanosync) [inline src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java]: Can probably reference AddReminderCommand's usage here

File: src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java [hunk 2]
```diff
@@ -1,49 +1,24 @@
...
-
-    @Override
-    public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof ReminderCommand // instanceof handles nulls
-                && toAdd.equals(((ReminderCommand) other).toAdd));
-    }
+            + PREFIX_MESSAGE + "MESSAGE "
+            + "\n" + " or " + "\n"
+            + COMMAND_WORD + " delete" + ": Remove a Reminder from MoneyGoWhere. "
```
Comment (Nanosync) [inline src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java]: Same for delete

File: src/main/java/seedu/moneygowhere/logic/commands/reminder/AddReminderCommand.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Reminder to MoneyGoWhere. "
+            + "Parameters: "
+            + PREFIX_DATE + "DATE "
+            + PREFIX_MESSAGE + "MESSAGE ";
+
+    public static final String MESSAGE_SUCCESS = "New Reminder added: %1$s";
+
+    private final Reminder toAdd;
```
Comment (Nanosync) [inline src/main/java/seedu/moneygowhere/logic/commands/reminder/AddReminderCommand.java]: Better to keep it as `reminder` or `value`

**Review by Nanosync — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/moneygowhere/logic/commands/reminder/AddReminderCommand.java (hunk 3 above)
Comment (minpyaemoe) [reply src/main/java/seedu/moneygowhere/logic/commands/reminder/AddReminderCommand.java]: i will change the variable name to reminderToAdd.

File: src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java (hunk 1 above)
Comment (minpyaemoe) [reply src/main/java/seedu/moneygowhere/logic/commands/ReminderCommand.java]: thanks for pointing out

**Review by Nanosync — APPROVED** [review]

**Comment by Nanosync** [comment]
Fixes #127

**Pull request merged**